jekyll/_cci2/security.md

```diff
@@ -52,6 +52,27 @@ A few different external services and technology integration points touch Circle
 
 - **iOS Builds** If you are paying to run iOS builds on CircleCI hardware your source code will be downloaded to a build box on our macOS fleet where it will be compiled and any tests will be run. Similar to our primary build containers that you control, the iOS builds we run are sandboxed such that they cannot be accessed.
 
+- **Docker** If you are using Docker images, refer to the public Docker [seccomp (security computing mode) profile](https://github.com/docker/engine/blob/e76380b67bcdeb289af66ec5d6412ea85063fc04/profiles/seccomp/default.json) for the Docker engine. CircleCI appends the following to the Docker default `seccomp` profile: 
+
+{% raw %}
+```
+[
+  {
+    "comment": "CIRCLE-16825 Allow create user namespaces",
```

Comment:
I wonder whether there's any value or confusion caused by leaving the jira reference here?

Reply:
oh, yeah, let's remove it